lib/spack/docs/build_systems/autotoolspackage.rst

```diff
@@ -0,0 +1,301 @@
+.. _autotoolspackage:
+
+----------------
+AutotoolsPackage
+----------------
+
+Autotools is a GNU build system that provides a build-script generator.
+By running the platform-independent ``./configure`` script that comes
+with the package, you can generate a platform-dependent Makefile.
+
+^^^^^^
+Phases
+^^^^^^
+
+The ``AutotoolsPackage`` base class comes with the following phases:
+
+#. ``autoreconf`` - generate the configure script
+#. ``configure`` - generate the Makefiles
+#. ``build`` - build the package
+#. ``install`` - install the package
+
+Most of the time, the ``autoreconf`` phase will do nothing, but if the
+package is missing a ``configure`` script, ``autoreconf`` will generate
+one for you.
+
+The other phases run:
+
+.. code-block:: console
+
+   $ ./configure --prefix=/path/to/installation/prefix
+   $ make
+   $ make check  # optional
+   $ make install
+   $ make installcheck  # optional
+
+
+Of course, you may need to add a few arguments to the ``./configure``
+line.
+
+^^^^^^^^^^^^^^^
+Important files
+^^^^^^^^^^^^^^^
+
+The most important file for an Autotools-based package is the ``configure``
+script. This script is automatically generated by Autotools and generates
+the appropriate Makefile when run.
+
+.. warning::
+
+   Watch out for fake Autotools packages!
+
+   Autotools is a very popular build system, and many people are used to the
+   classic steps to install a package:
+
+   .. code-block:: console
+
+      $ ./configure
+      $ make
+      $ make install
+
+
+   For this reason, some developers will write their own ``configure``
+   scripts that have nothing to do with Autotools. These packages may
+   not accept the same flags as other Autotools packages, so it is
+   better to use the ``Package`` base class and create a
+   :ref:`custom build system <custompackage>`. You can tell if a package
+   uses Autotools by running ``./configure --help`` and comparing the output
+   to other known Autotools packages. You should also look for files like:
+
+   * ``configure.ac``
+   * ``configure.in``
+   * ``Makefile.am``
+
+   Packages that don't use Autotools aren't likely to have these files.
+
+^^^^^^^^^^^^^^^^^^^^^^^^^
+Build system dependencies
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Whether or not your package requires Autotools to install depends on
+how the source code is distributed. Most of the time, when developers
+distribute tarballs, they will already contain the ``configure`` script
+necessary for installation. If this is the case, your package does not
+require any Autotools dependencies.
+
+However, a basic rule of version control systems is to never commit
```

Comment:
I think this is a bit more detail than is required for a Spack user. IMO you could refocus the conversation:

> Some packages use higher-level specification files (e.g. ` ``configure.ac`` ` and ` ``Makefile.am`` `) to generate a configure script. In this case, Spack will automatically run `autoreconf` to generate the configure script, but you need to add some dependencies...

That also sidesteps issues related to properly citing the .svg file (although that by itself is not a good reason to omit it).